### DISCUSSION TITLE
Format height for PerspectiveWidget

### DISCUSSION THREAD

**Discussion by dhirschfeld** [perspective-dev/perspective#1462] [Q&A] [answered]
Hi,
I'm trying to figure out how to configure the height of the displayed `PerspectiveWidget`.

IIUC `ipywidgets` have a [`layout`](https://ipywidgets.readthedocs.io/en/latest/examples/Widget%20Styling.html#Layout-and-Styling-of-Jupyter-widgets) property which allows you to configure some basic css but `Perspective Widget` doesn't seem to support that. Is it possible to configure the height of the `PerspectiveWidget`?

**Comment by sc1f** [comment, marked as answer]
I'll look into why the `layout` properties aren't being used by `PerspectiveWidget` - in the meantime you can use `display` and `HTML`:

```python
from IPython import display, HTML
display(HTML("<style>perspective-viewer { height: 600px !important; }</style>"))
```

it's a little hacky but it's guaranteed to work.

**Reply by sc1f** [reply]
Quick update on this - `layout` did not work because our Python widget did not inherit from `DOMWidget`. I've fixed this in #1466 and it should be in the next release.
